openapi3/refs.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/getkin/kin-openapi/jsoninfo"
 )
 
+func marshalRef(v interface{}, r string) ([]byte, error) {
```

Comment:
what is the purpose of this function?

Reply:
see above comment (initial version had a store/lookup mechanism but it was not sufficient. This is still a place holder for future work (although it's not clear to me yet how to go about doing this)

Reply:
ok, maybe make a TODO comment to this effect so I can keep track.